lib/iris/tests/unit/plot/test_2d_coords.py

```diff
@@ -22,29 +22,34 @@
 # Import iris.tests first so that some things can be initialised before
 # importing anything else.
 import iris.tests as tests
-import iris.coords as coords
 
 import numpy.ma as ma
 
+import iris.coords as coords
 from iris.tests.stock import simple_2d_w_multidim_coords as cube_2dcoords
 from iris.tests.stock import simple_3d_w_multidim_coords as cube3d_2dcoords
-from iris.tests.stock import lat_lon_cube
+from iris.tests.stock import sample_2d_latlons
+from iris.tests.stock import make_bounds_discontiguous_at_point
 
-from orca_utils.plot_testing import testdata_2d_coords as testdata
 
 if tests.MPL_AVAILABLE:
     import iris.plot as iplt
 
 
+def full2d_global():
+    return sample_2d_latlons(transformed=True)
+
+
+@tests.skip_data
 class Test_2d_coords_plot_defn_bound_mode(tests.IrisTest):
     def setUp(self):
         self.multidim_cube = cube_2dcoords()
         self.overspan_cube = cube3d_2dcoords()
 
         # latlon_2d is a cube with 2d coords, 4 bounds per point,
         # discontiguities in the bounds but masked data at the discontiguities.
-        self.latlon_2d = testdata.full2d_global()
-        testdata.make_bounds_discontiguous_at_point(self.latlon_2d, 2, 2)
+        self.latlon_2d = full2d_global()
+        make_bounds_discontiguous_at_point(self.latlon_2d, 2, 2)
```

Comment:
Only remaining usage of this routine : see https://github.com/SciTools/iris/pull/3127#discussion_r207522639 
Can remove later